rdf-delta-cmds/src/main/java/delta/server/ServerBuildLib.java

```diff
@@ -197,7 +197,7 @@ private static LocalServerConfig setupS3(DeltaServerConfig config, LocalServerCo
     }
 
     // --> DeltaServer.start()
-    private static DeltaServer buildServer(int port, Supplier<LocalServerConfig> startup) {
+    private static DeltaServer buildServer(Integer port, Supplier<LocalServerConfig> startup) {
```

Comment:
Need to change to `Integer` to allow a `null` value to be passed when `jettyConf` is supplied.